src/app/api/routes/alerts.py

```diff
@@ -112,22 +115,26 @@ async def fetch_alerts(requester=Security(get_current_access,
 
 
 @router.put("/{alert_id}/", response_model=AlertOut, summary="Update information about a specific alert")
-async def update_alert(
-    payload: AlertIn,
-    alert_id: int = Path(..., gt=0),
-    _=Security(get_current_access, scopes=[AccessType.admin])
-):
+async def update_alert(payload: AlertIn,
+                       alert_id: int = Path(..., gt=0),
+                       requester=Security(get_current_access, scopes=[AccessType.admin, AccessType.user])):
     """
     Based on a alert_id, updates information about the specified alert
     """
+    requested_group_id = await get_entity_group_id(alerts, alert_id)
+    await check_group_update(requester.id, requested_group_id)
     return await crud.update_entry(alerts, payload, alert_id)
```

Comment:
same here

Reply:
same suggestion as above roughly, not sure we can really speed up the process